src/amo/css/OverallRating.scss

```diff
@@ -0,0 +1,40 @@
+.OverallRating {
+
+  form {
+    display: flex;
+    justify-content: center;
+
+    input {
+      appearance: none;
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    label {
+      font-size: 0.8em;
+      font-weight: bold;
+      cursor: pointer;
+      display: block;
+      text-align: center;
+      padding-top: 90px;
+      width: 120px;
+      height: 100px;
+      background-size: 85px;
+      background-position: center;
+      background-repeat: no-repeat;
+    }
+
+    label.love-it {
+      background-image: url('../img/ratings/smiley-love-it.svg');
+    }
+
+    label.it-is-ok {
+      background-image: url('../img/ratings/smiley-it-is-ok.svg');
+    }
+
+    label.huh {
+      background-image: url('../img/ratings/smiley-huh.svg');
```

Comment:
We should file something on this to look at using svgo later to crush the svgs (probably something we can do via webpack and also we might want to consider these 3 being a single sprite with something like svg_sprite too although if they're small enough to be data/uris then we won't need to.